tests/nlu/base/test_evaluation.py

```diff
@@ -320,6 +326,59 @@ def test_intent_evaluation_report(tmpdir_factory):
     assert result["predictions"][0] == prediction
 
 
+def test_intent_evaluation_report_large(tmpdir_factory):
+    path = tmpdir_factory.mktemp("evaluation").strpath
+    report_folder = os.path.join(path, "reports")
+    report_filename = os.path.join(report_folder, "intent_report.json")
+
+    rasa.utils.io.create_directory(report_folder)
+
+    def correct(label):
+        return IntentEvaluationResult(label, label, "", 1.0)
+
+    def incorrect(label, _label):
+        return IntentEvaluationResult(label, _label, "", 1.0)
+
+    a_results = [correct("A")] * 10
+    b_results = [correct("B")] * 7 + [incorrect("B", "C")] * 3
+    c_results = [correct("C")] * 3 + [incorrect("C", "D")] + [incorrect("C", "E")]
+    d_results = [correct("D")] * 29 + [incorrect("D", "B")] * 3
+    e_results = [incorrect("E", "C")] * 5 + [incorrect("E", "")] * 5
+
+    intent_results = a_results + b_results + c_results + d_results + e_results
+
+    evaluate_intents(
+        intent_results,
+        report_folder,
+        successes=False,
+        errors=False,
+        confmat_filename=None,
+        intent_hist_filename=None,
+    )
+
+    report = json.loads(rasa.utils.io.read_file(report_filename))
```

Comment:
or fix `rasa.utils.io.read_file` (same as above)
```suggestion
    report = json.loads(rasa.utils.io.read_file(str(report_filename)))
```